controllers/managedfusionoffering_controller.go

```diff
@@ -129,6 +136,14 @@ func (r *ManagedFusionOfferingReconciler) reconcilePhases() (reconcile.Result, e
 					return ctrl.Result{}, fmt.Errorf("failed to remove finalizer from %s secret: %v", managedFusionSecretName, err)
 				}
 			}
+			r.Log.Info(fmt.Sprintf("deleting %s namespace", r.namespace))
```

Comment:
- a couple of lines above, log seems to be incorrect, could you pls replace `secret` with `offering`?

Reply:
Updated